showmore_dyn/showmore_dyn.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Name: Showmore Dynamic
+ * Description: Dynamically limits height of posts
+ * Version: 1.0
+ * Author: Christian Wiwie
+ *
+ */
+
+use Friendica\App;
+use Friendica\Core\Hook;
+use Friendica\Core\L10n;
+use Friendica\Core\Logger;
+use Friendica\Core\Renderer;
+use Friendica\Database\DBA;
+use Friendica\DI;
+
+function showmore_dyn_install() {
+	Hook::register('page_end', 'addon/showmore_dyn.php/showmore_dyn.php', 'showmore_dyn_script');
+	Hook::register('head'  , __FILE__, 'showmore_dyn_head');
+	Hook::register('footer', __FILE__, 'showmore_dyn_footer');
+	Hook::register('addon_settings', 'addon/showmore_dyn/showmore_dyn.php', 'showmore_dyn_settings');
+        Hook::register('addon_settings_post', 'addon/showmore_dyn/showmore_dyn.php', 'showmore_dyn_settings_post');
+}
+
+function showmore_dyn_uninstall()
```

Comment:
Please remove the `_uninstall()` method as it doesn't do anything beyond unregistering hooks, which is automatically done on addon uninstall.